autotest/common_regression.py

```diff
@@ -612,60 +619,49 @@ def get_mf6_mshape(disfile):
     return mshape
 
 
-def get_mf6_files(mfnamefile):
-    """Return a list of all the MODFLOW 6 input and output files in this model.
+def get_mf6_files(namefile):
+    """Get all MODFLOW 6 input and output files in this simulation.
 
     Parameters
     ----------
-    mfnamefile : str
+    namefile : pathlike
         path to the MODFLOW 6 simulation name file
 
     Returns
     -------
-    filelist : list
-        list of MODFLOW 6 input files in a simulation
-    outplist : list
-        list of MODFLOW 6 output files in a simulation
-
+    A tuple of lists of paths (input files, output files)
     """
 
-    srcdir = os.path.dirname(mfnamefile)
-    filelist = []
-    outplist = []
-
-    filekeys = ["TDIS6", "GWF6", "GWT", "GWF6-GWF6", "GWF-GWT", "IMS6"]
-    namefilekeys = ["GWF6", "GWT"]
-    namefiles = []
-
-    with open(mfnamefile) as f:
-
-        # Read line and skip comments
-        lines = f.readlines()
-
-    for line in lines:
-
+    srcdir = os.path.dirname(namefile)
+    mdl_files = []
+    pkg_files = []
+    out_files = []
+    pkg_keys = ["TDIS6", "GWF6", "GWT", "GWF6-GWF6", "GWF-GWT", "IMS6"]
+    mdl_keys = ["GWF6", "GWT"]
```

Comment:
Should `GWT` be `GWT6` like `GWF6`. Also maybe it would be clearer if `mdl_keys` was `model_keys`.

Reply:
fixed